### DISCUSSION TITLE
How to start

### DISCUSSION THREAD

**Discussion by JuSarmi** [coqui-ai/TTS#3975] [General]
Hi  I am Sebastian, I want to create a character from some voice recordings to make a translation, Do you know what guides/videos/text should I start, thanks for reading.

**Comment by eginhard** [comment]
https://coqui-tts.readthedocs.io/en/latest/tutorial_for_nervous_beginners.html

**Comment by joshgura** [comment]
so this is a command line program? or is there a GUI as well?

ultimately i want to create a tts model that i can use in some program that can load a tts model. but i thought i remember seeing there was a coqui with a webui or something.

**Comment by Ahtisham-1214** [comment]
I am creating a Text-to-speech project in Python where user will give English text as input and in response will get a voice accordingly. I want to use any Coqui TTS model, I just want to make it work. The documentation is a little confusing so please help me out.
@eginhard @xeb @joshgura @akx
